src/app/games/quarto/quarto.component.ts

```diff
@@ -1,29 +1,27 @@
 import { Component } from '@angular/core';
 import { QuartoMove } from './QuartoMove';
-import { QuartoPartSlice } from './QuartoPartSlice';
+import { QuartoState } from './QuartoState';
 import { QuartoRules } from './QuartoRules';
 import { QuartoMinimax } from './QuartoMinimax';
 import { QuartoPiece } from './QuartoPiece';
-import { AbstractGameComponent } from '../../components/game-components/abstract-game-component/AbstractGameComponent';
 import { Coord } from 'src/app/jscaip/Coord';
 import { MGPValidation } from 'src/app/utils/MGPValidation';
-import { MoveEncoder } from 'src/app/jscaip/Encoder';
 import { MessageDisplayer } from 'src/app/services/message-displayer/MessageDisplayer';
 import { RulesFailure } from 'src/app/jscaip/RulesFailure';
-import { TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
 import { quartoTutorial } from './QuartoTutorial';
+import { RectangularGameComponent } from 'src/app/components/game-components/rectangular-game-component/RectangularGameComponent';
 
 @Component({
     selector: 'app-quarto',
     templateUrl: './quarto.component.html',
     styleUrls: ['../../components/game-components/abstract-game-component/abstract-game-component.css'],
 })
-export class QuartoComponent extends AbstractGameComponent<QuartoMove, QuartoPartSlice> {
-
-    public rules: QuartoRules = new QuartoRules(QuartoPartSlice);
-
-    public CASE_SIZE: number = 100;
```

Comment:
à ajouter au constructeur: this.CASE_SIZE = 100

Reply:
Fait !